src/stability_sdk/client.py

```diff
@@ -114,7 +114,7 @@ def __init__(
         self,
         host: str = "grpc.stability.ai:443",
         key: str = "",
-        engine: str = "stable-diffusion-v1-5",
+        engine: str = "stable-diffusion-v2-0",
```

Comment:
Grabbed the new engine ID from here: https://github.com/Stability-AI/stability-bot/commit/733d98d6559b73d23f54ed4bda62401510b9251d

Following error thrown when I tried running the demo notebook:

```python
_MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.NOT_FOUND
	details = "The specified engine (ID stable-diffusion-v2-0) was not found."
	debug_error_string = "UNKNOWN:Error received from peer ipv4:104.18.20.212:443 {grpc_message:"The specified engine (ID stable-diffusion-v2-0) was not found.", grpc_status:5, created_time:"2022-11-24T18:13:31.997226136+00:00"}"
>
```

Has the new engine just not been deployed yet? or is this identifier string incorrect and needs to be fixed both here and the stability-bot repo? @palp @wbrown